src/controllers/user/index.js

```diff
@@ -0,0 +1,87 @@
+import { CognitoUser, AuthenticationDetails } from 'amazon-cognito-identity-js'
```

Comment:
Let's move all cognito logic to a separate file (src/controllers/user/cognito.js or src/services/cognito.js would be ok, whatever you like better).

Reply:
Done